### ISSUE TITLE
Is it possible to reduce the widget of the active option background view?

### ISSUE THREAD

**Issue by prakashraman** [xaviermerino/XMSegmentedControl#35]
Is it possible to reduce the widget of the active option background view?

I am basically trying to create the following.

<img width="609" alt="segment-control" src="https://user-images.githubusercontent.com/36934/27648546-82f4090c-5c4c-11e7-8d0f-4db591dd1dd1.png">

Is this possible using the library?

**Comment by xaviermerino** [comment]
Yes by handling the radius and corner options of the active background view you can make it a circle. There is no direct way of doing it via the exposed functions but adding a couple of lines to the source code will do.

**Comment by prakashraman** [comment]
Ah, brilliant. Yes, that sounds simple enough. Let me see how to reference the active-background-view. In case I don't find - I shall seek your help again.

Thanks!

**Comment by prakashraman** [comment]
@xaviermerino  I guess I will have to access the property. "highlightview" right?

Also - it seems to have an access control of "fileprivate". Will I be able to access the property from my ViewController?
